### PR TITLE
Warn about undelivered notifications

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: src/NotificationsApp.vue [hunk 1]
```diff
@@ -32,6 +33,15 @@
 					<transition-group class="notification-wrapper"
 						name="list"
 						tag="ul">
+						<Notification v-if="hasThrottledPushNotifications"
+							:key="-2016"
+							datetime="warning"
```
Comment (PVince81) [inline src/NotificationsApp.vue:38]: datetime=warning ?

File: src/NotificationsApp.vue [hunk 2]
```diff
@@ -133,13 +162,47 @@ export default {
 			}
 			return false
 		},
+		isOrangeThemed() {
```
Comment (PVince81) [inline src/NotificationsApp.vue:165]: am surprised this is even a thing :sweat_smile:

**Review by PVince81 — APPROVED** [review]
:+1: code looks fine

File: src/NotificationsApp.vue (hunk 1 above)
Comment (nickvergessen) [reply src/NotificationsApp.vue:38]: Yeah, its a hack to identify this notification inside the component...

**Pull request merged**